build-images.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
Is this for debugging? Should we check it in?

Reply:
It is useful when doing local testing. It isn't required by the CI build.

Reply:
I am ok w/ adding this - do you mind adding comment to that effect at the top though?